src/rgw/rgw_cr_rados.h

```diff
@@ -57,7 +57,7 @@ class RGWAsyncRadosRequest : public RefCountedObject {
 
 class RGWAsyncRadosProcessor {
   deque<RGWAsyncRadosRequest *> m_req_queue;
-  atomic_t going_down;
+  std::atomic<unsigned> going_down = { 0 };
```

Comment:
please rebase - this change was already merged in https://github.com/ceph/ceph/pull/14839